### PR TITLE
[PB] Column component

### PR DESCRIPTION
**Pull request by eug123**
## Description
Introduce Column component for Page Builder Column content type

## Related Issue
https://jira.corp.magento.com/browse/PB-8

## Performance Results
https://wiki.corp.magento.com/display/COREENG/Column

## Verification Steps
1. Check that each property for each content type appearance renders correctly on PWA Venia storefront in desktop and mobile.
2. Compare Venia PWA rendering of content type with Luma to ensure the experience is the same or better for each property of each appearance in desktop and mobile.
3. Check each supported browser and on mobile that is supported according to PWA documentation.
4. Acceptable performance has been validated

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/pwa-studio?pullRequest=1726) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/pwa-studio?pullRequest=1726) before we can accept your contribution.<br/>

File: packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/__tests__/configAggregator.spec.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            backgroundAttachment: 'scroll',
+            backgroundColor: 'rgb(193, 193, 193)',
+            backgroundPosition: 'left top',
+            backgroundRepeat: false,
+            backgroundSize: 'cover',
+            border: 'solid',
+            borderColor: 'rgb(0, 0, 0)',
+            borderRadius: '10px',
+            borderWidth: '1px',
+            cssClasses: ['pagebuilder-column', 'myCssColumnClass'],
```
Comment (davemacaulay) [inline packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/__tests__/configAggregator.spec.js]: We don't currently verify advanced properties within our configAggregator tests, as these utility methods are already tested within our implementation.

See https://github.com/magento/pwa-studio/pull/1698/files#diff-f7624a8cda70ade6382454adf8f2d960 as an example, we only validate properties which are not generated from the utility methods.

File: packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/configAggregator.js [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
         width: node.style.width,
-        ...getVerticalAlignment(node),
         backgroundColor: node.style.backgroundColor,
+        ...getAdvanced(node),
         ...getBackgroundImages(node),
-        cssClasses: node.getAttribute('class')
-            ? node.getAttribute('class')
-            : null,
-        ...getAdvanced(node)
+        ...getBorder(node),
```
Comment (davemacaulay) [inline packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/configAggregator.js]: `getAdvanced` internally calls `getBorder`, `getCssClasses` etc. So you only need `getAdvanced` or the others.

Please see: https://github.com/magento/pwa-studio/blob/5b50350c31fa5607d5d4e61db39aab83b32d30c6/packages/venia-ui/lib/components/RichContent/PageBuilder/utils.js#L84-L92

File: pwa-devdocs/scripts/create-reference-docs/config/venia-concept/index.js [hunk 3]
```diff
@@ -2,5 +2,9 @@ module.exports = [
   {
     target: 'venia-ui/lib/components/ToastContainer/toastContainer.js',
     type: 'function'
+  },
+  {
+    target: 'venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/column.js',
+    type: 'function'
```
Comment (davemacaulay) [inline pwa-devdocs/scripts/create-reference-docs/config/venia-concept/index.js]: You'll also need to add a reference to `columnGroup.js` in this file.

File: packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/column.js [hunk 4]
```diff
@@ -114,39 +126,73 @@ const Column = props => {
...
+/**
+ * Props for {@link Column}
+ *
+ * @typedef props
+ *
+ * @property {String} appearance Converts to CSS align-self sub-property of the flexbox item
+ * @property {String} backgroundAttachment CSS background-attachment property
+ * @property {String} backgroundColor CSS background-color property
+ * @property {String} backgroundPosition CSS background-position property
+ * @property {String} backgroundRepeat CSS background-repeat property
```
Comment (davemacaulay) [inline packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/column.js]: backgroundRepeat should be boolean

**Pull request merged**